### ISSUE TITLE
Disabling terrain

### ISSUE THREAD

**Issue by mghildiy** [Melown/vts-browser-js#110]
Hi,

I want to render only the geometry data embedded in geoJson. 
VTS seems to render default terrain too.

Can this be diabled?

With regards
Manish

**Comment by DavidLevinsky** [comment]
Hi,

yes, you can remove surface from the map view.  You can obtain map view by this function:
```
var mapView = map.getView();
```
It will return something like this:
```
mapView =  {
  "surfaces": {
    "some-surface": []
  },
  "freeLayers": {
    "some-free-layer": {}
  }
}
```
then you remove surfaces from the map view object:
```
mapView = {
  "surfaces": {},
  "freeLayers": {
    "some-free-layer": {}
  }
}
```
and apply it to the map:
```
map.setView(mapView);
```

**Comment by mghildiy** [comment]
Thanks a lot for the inputs.

**Comment by mghildiy** [comment]
Unfortunately this didn't work.
My browser just goes black with this change.

**Comment by DavidLevinsky** [comment]
There was a bug which cased reported behavior. Bugfix will be released during next week.

**Comment by DavidLevinsky** [comment]
It should be fixed in new version (2.16.0)
